.github/workflows/release.yml

```diff
@@ -12,14 +12,16 @@ on:
           - patch
           - minor
           - major
+      baseTag:
+        description: 'base release tag'
```

Comment:
adding it as an input doesn't do much if it isn't passed to the action :) let's remove it, it's not needed